tests/python/gaia-ui-tests/gaiatest/brick/manifest.ini

```diff
@@ -0,0 +1,6 @@
+[DEFAULT]
+external = false
+online = false
+
+[test_brick_verification.py]
+smoketest = true
```

Comment:
This shouldn't be run as part of the smoketests. That's only really a concept inside the tests/functional suite. Your test will always be run with full manifest.

Really, you don't need a manifest, since we'd only run this one test.